app/controllers/idv/personal_key_controller.rb

```diff
@@ -37,6 +37,8 @@ def next_step
         idv_come_back_later_url
       elsif in_person_enrollment?
         idv_in_person_ready_to_verify_url
+      elsif device_profiling_failed?
+        idv_come_back_later_url
```

Comment:
we'll change this once we have real content, right?

Reply:
I feel like this PR should also include a new stub page like `/verify/setup_error` that we'll use for the failure messaging (it can just say something like "Please contact support to continue account setup" or something)

Reply:
yep